client/navigation/components/container/index.js

```diff
@@ -84,7 +84,17 @@ const Container = ( { menuItems } ) => {
 			<Navigation
 				activeItem={ activeItem ? activeItem.id : null }
 				activeMenu={ activeLevel }
-				onActivateMenu={ setActiveLevel }
+				onActivateMenu={ ( { menuId } ) => {
+					const navDomRef = document.querySelector(
```

Comment:
Probably a little bit edge case, but it would be nice to use `useRef()` to get this element instead of a query selector.  We might need to wrap the `Navigation` with a class like `woocommerce-navigation__wrapper` to make that work though.

Reply:
Yeah, I started to go for a `useRef()` but realized that it would require a wrapper/etc as you mentioned. Wasn't sure it was worth the extra boilerplate. Is the main concern that the targeted class could change @joshuatf ?

Reply:
There is the possibility of the class changing, but there's also the possibility that the `Navigation` component is used multiple times across a page.  We don't really have a use case for this (yet), but if there are multiple navigation components, the added styling for scroll behavior on the class and this `scrollTop` would affect all of the components and not just this one.

I think adding a wrapping class is little overhead to guard against future conflicts.